pkg/skbn/delete.go

```diff
@@ -0,0 +1,155 @@
+package skbn
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math"
+	"path/filepath"
+
+	"github.com/Cognologix/skbn/pkg/utils"
+)
+
+// Delete files from path
+func Delete(path string, parallel int) error {
+	prefix, path := utils.SplitInTwo(path, "://")
+
+	err := TestImplementationsExistForDelete(prefix)
+	if err != nil {
+		return err
+	}
+	client, err := GetClient(prefix, path)
+	if err != nil {
+		return err
+	}
+	relativePaths, err := GetListOfFiles(client, prefix, path)
+	if err != nil {
+		return err
+	}
+
+	var absolutePaths []string
+	for _, relativePath := range relativePaths {
+		absolutePaths = append(absolutePaths, filepath.Join(path, relativePath))
+	}
```

Comment:
```suggestion
	relativePaths, err := GetListOfFiles(client, prefix, path)
	if err != nil {
		return err
	}

	var absolutePaths []string
	for _, relativePath := range relativePaths {
		absolutePaths = append(absolutePaths, filepath.Join(path, relativePath))
	}
```

Relative path logic should be in separate function.